### PR TITLE
VZ-9116-1: Add opensearch-operator charts

### PR DESCRIPTION
**Pull request by saketmht**
Added opster opensearch-operator charts

### CODE REVIEW

**Comment by adiforluls** [comment]
You're uptaking the upstream chart as-is right?

**Comment by saketmht** [comment]
> You're uptaking the upstream chart as-is right?

Yes, for this PR, there are no modifications to the upstream charts.

**Review by adiforluls — DISMISSED** [review]

**Review by IshaGirdhar — APPROVED** [review]

File: platform-operator/thirdparty/charts/README.md [hunk 1]
```diff
@@ -220,3 +220,14 @@ rm -rf prometheus-community/prometheus-node-exporter
 helm fetch prometheus-community/prometheus-node-exporter --untar=true --version=${PROMETHEUS_NODE_EXPORTER_CHART_VERSION}
 ```
 
+### Opensearch Operator
```
Comment (kminder) [inline platform-operator/thirdparty/charts/README.md:223]: This should be OpenSearch, right?

Comment (saketmht) [reply platform-operator/thirdparty/charts/README.md]: Fixed

**Review by gaurav-05 — APPROVED** [review]

**Pull request merged**